pkg/opcfg/config.go

```diff
@@ -34,6 +34,16 @@ func GetIsWebhookEnabled() bool {
 	return lookupBoolEnvVar("WEBHOOKS_ENABLED", envMustExist)
 }
 
+// GetBroadcasterBurstSize returns the customizable burst size for broadcaster.
+func GetBroadcasterBurstSize() int {
+	burstSize := lookupIntEnvVar("BROADCASTER_BURST_SIZE", envCanNotExist)
+	if burstSize < 25 {
```

Comment:
Minor: We can make a constant DEFAULT_BURST_SIZE = 25 for maintaining the int, and we don't need an else block here, we can do:
`if burstSize >= DEFAULT_BURST_SIZE {
    return burstSize
}
return DEFAULT_BURST_SIZE 
`

Reply:
made the change.